### DISCUSSION TITLE
frontend folder having a lot of type errors

### DISCUSSION THREAD

**Discussion by cocomatias** [langflow-ai/langflow#2464] [General]
Im starting this discussion since on the company im working on we're trying to use this on production and there's a lot of type errors everywhere. Automatic testing for this is pretty easy since it's just a `npm run type-check`, so not sure why that's not implemented on the PR checks. Im doing this on my side on my company ts repositories commits using husky so nobody can even push changes anywhere without initial testings.

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
